lazybytez/jojo-discord-bot
# Rework component registration system

## Description
Rework the component registration system to better align with standards.
It also greatly improves API design as we now can also have access to our components in the API.

## Related issue
Resolves #79 

## How can this be tested?
 - Check if bot start and stop works
 - Test if slash commands works (enabled / disabled component)
 - Test if event handlers work (example: ping pong). Also check enabled / disabled components
